### ISSUE TITLE
Is it possible train models in windows?

### ISSUE THREAD

**Issue by omid-ghozatlou** [junyanz/pytorch-CycleGAN-and-pix2pix#1239]
I don't have Linux and want to run this code in windows with GPU. Is it possible to run it or Do I have to change something?

**Comment by rickkk856** [comment]
I'm a windows user and I have already used this repo... just follow the installation instructions, and make sure you installed all de dependencies... 

I just couldn't figure out yet how to plot loss from loss.txt and how to use the combine_A_and_B.py

**Comment by rickkk856** [comment]
combining datasets A + B should have a change in main see --> [https://medium.com/@mackie__m/running-a-cifar-10-image-classifier-on-windows-with-pytorch-9094e29089cd](This Link)

**Comment by rickkk856** [comment]
combining datasets A + B should have a change in main see --> [this link](https://medium.com/@mackie__m/running-a-cifar-10-image-classifier-on-windows-with-pytorch-9094e29089cd)

**Comment by omid-ghozatlou** [comment]
I have implemented it on Windows I got just a few warnings about visdom. It works also on Windows.

**Comment by rickkk856** [comment]
#Update - It seems if you train on windows, specially using `pytorch==1.4.0 torchvision==0.5.0 cudatoolkit=10.1` you won't be able  to reproduce results on google colab...

I'm not sure if it's a Torch / Torchvision / Cuda incompatibility error
OR if it's a windows-unbutu error.

**Comment by ApoorvaSuresh** [comment]
@rickkk856 @omid-ghozatlou could you please let me know how you downloaded the datasets? (the linux implementation uses bash, but how is it done in windows?)
Thank you :)

**Comment by rickkk856** [comment]
@ApoorvaSuresh I'm using custom datasets, however, you can find it on the code or search for it on other sources like [Kaggle - Labels2Facades](https://www.kaggle.com/balraj98/facades-dataset?select=trainA)  you can also find them [Here](http://efrosgans.eecs.berkeley.edu/pix2pix/datasets/)

**Comment by ApoorvaSuresh** [comment]
@rickkk856 Thank you! Did the visdom server work for you? I always get "Could not connect to Visdom server"

**Comment by rickkk856** [comment]
@ApoorvaSuresh  I could also connect to visdom, but I had to run on one CMD as administrator `python -m visdom.server` and then run `python train.py` in a different cmd... if you still can't do it, try `pip install -r requirements.txt` again, or install a different version of visdom server.
